aws-cache-helpers/src/internal/utils/dynamodb.ts

```diff
@@ -0,0 +1,38 @@
+import {unmarshall} from '@aws-sdk/util-dynamodb';
+import {AttributeValue} from 'aws-lambda';
+
```

Comment:
these import warnings are interesting.

Reply:
ugh yeah this was really anoying was messing with this for a bit maybe we can pair on this in follow up pr. This one reasons want to just get v0 published and just make sure it works in consuming code base as well.  The actual package is `'@types/aws-lambda'` but Intelij was yelling at me to rename to `aws-lambda`. 
 
<img width="1019" alt="Screenshot 2023-01-12 at 10 37 17 AM" src="https://user-images.githubusercontent.com/5491827/212150970-de4b06fd-7fa6-497e-89af-d250a9c9c88c.png">

Reply:
fast-follow is okay